### PR TITLE
Add tests for utils and post handler

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: script/worker_test [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+# Build the worker
+
+# Stop on errors
+set -e
+
+cd "$(dirname "$0")/.."
+
+if [ ! -d node_modules ]; then
```
Comment (balloob) [inline script/worker_test:9]: Locally this is not enough as package-lock might have changed. It's better to just not do this, and require people to call `script/bootstrap` in their flows when they need it.

File: worker/src/handlers/post.ts [hunk 2]
```diff
@@ -69,7 +72,7 @@ async function handlePost(request: Request, sentry: Toucan): Promise<Response> {
 
   delete stored.value.last_write;
 
-  if (!deepEqual(stored.value, incomingPayload)) {
+  if (!deepEqual(stored.value, JSON.parse(JSON.stringify(incomingPayload)))) {
```
Comment (balloob) [inline worker/src/handlers/post.ts:75]: why stringify and parse?

Comment (ludeeus) [reply worker/src/handlers/post.ts]: Because superstruct adds undefined optional keys to the object, which are not present when stringifying, added comment

File: worker/tests/handlers/post.spec.ts [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    expect(MockKV.put).toBeCalledTimes(1);
+  });
+
+  it("Data changed", async () => {
+    MockKV.getWithMetadata = jest.fn(async () => ({
+      value: { ...BASE_PAYLOAD },
+    }));
+    MockRequest.json = async () =>
+      JSON.parse(
+        JSON.stringify({
```
Comment (balloob) [inline worker/tests/handlers/post.spec.ts:73]: Why? It's in a function so it's new each time ?

File: worker/tests/utils/average.spec.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { average } from "../../src/utils/average";
+
+it("test", function () {
```
Comment (balloob) [inline worker/tests/utils/average.spec.ts:3]: Try testing the edge cases. What is the average of [0, 1], a float or an int.

**Review by balloob — APPROVED** [review]
